src/toast/ops/totalconvolve.py

```diff
@@ -277,6 +277,13 @@ def _exec(self, data, detectors=None, **kwargs):
             beam = self.get_beam(beam_file, lmax, mmax, det, verbose)
 
             theta, phi, psi, psi_pol = self.get_pointing(data, det, verbose)
+            np.savez(
```

Comment:
thanks @mreineck this is amazing! I am surprised too that the tests pass! 
we can comment  this part that saves the angles , it was left from previous PR #534 .